mslib/mswms/dataaccess.py

```diff
@@ -391,6 +391,8 @@ def setup(self):
                     logging.error("Skipping file '%s' (%s: %s)", filename, type(ex), ex)
                     continue
                 self._file_cache[filename] = (mtime, content)
+                if content["vert_type"] not in self._elevations:
+                    self._elevations[content["vert_type"]] = content["elevations"]
```

Comment:
nice catch. did you run into an error or simply see that this is wrong?

Reply:
I ran into an error trying to mock another data directory. It didn't work out in the end but at least the error is caught.